documentation/ci-fix.md

```diff
@@ -44,3 +44,28 @@ oav validate-spec <swagger-spec-path>
 Please see [readme](https://github.com/Azure/oav/blob/master/README.md) for how to install or run tool in details.
 Or you can run it in [OpenAPI Hub](https://portal.azure-devex-tools.com/tools/static-validation/static/errors/default)
 Refer to [Semantic and Model Violations Reference](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Semantic-and-Model-Violations-Reference.md) for detailed description of validations and how-to-fix guidance.
+
+## Breaking Change Check
+
+run oad locally (the breaking change is reported by oad tool):
+```
+npm install -g oad
+oad compare <old-spec-path> <new-spec-path> 
+```
+Please see [readme](https://github.com/Azure/openapi-diff/blob/master/README.md) for how to install or run tool in details.
+Or you can run it in [OpenAPI Hub](https://portal.azure-devex-tools.com/tools/diff).
```

Comment:
Pls remove this link to openapi hub. it is not for breaking change

Reply:
actually they all use oad to diff, I don't find anything different

Reply:
In openapi hub, it is just a compare tool. not for breaking change you can try it, then you can understand why

Reply:
I have try it , the compare tool  is also oad,  the breaking change is based on result of oad

Reply:
Ok, got your mean, thanks Jianye